rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/RenderApplicationPieChartDirective.java

```diff
@@ -103,31 +103,31 @@ private void drawPie(Writer writer, Map<String, Integer> data, String elementID)
 
         String dataVarName = "data_" + elementID;
         writer.append("<script type='text/javascript'>");
-        writer.append("\nWINDUP_PACKAGE_PIE_DATA = typeof(WINDUP_PACKAGE_PIE_DATA) == 'undefined' ? {} : WINDUP_PACKAGE_PIE_DATA;");
-        writer.append("\nWINDUP_PACKAGE_PIE_DATA['").append(elementID).append("'] = [];");
+        writer.append(System.lineSeparator()+" WINDUP_PACKAGE_PIE_DATA = typeof(WINDUP_PACKAGE_PIE_DATA) == 'undefined' ? {} : WINDUP_PACKAGE_PIE_DATA;");
+        writer.append(System.lineSeparator()+" WINDUP_PACKAGE_PIE_DATA['").append(elementID).append("'] = [];");
         for (PieSort p : pieList)
         {
-            writer.append("\nWINDUP_PACKAGE_PIE_DATA['").append(elementID).append("'].push({label: '" + p.label + "', data: ")
+            writer.append(System.lineSeparator()+" WINDUP_PACKAGE_PIE_DATA['").append(elementID).append("'].push({label: '" + p.label + "', data: ")
                         .append(p.value.toString()).append("});");
```

Comment:
Why the extra space here? Omissions like this make me wonder how many bugs were introduced.

Reply:
I did mass find and replace, and in some cases it doesn't matter if there is additional or missing space(s). This code should generate javascript variable with content ;)